ruma-client-api/src/r0/backup.rs

```diff
@@ -1,12 +1,21 @@
 //! Endpoints for server-side key backups.
 
-pub mod add_backup_keys;
 pub mod create_backup;
+pub mod delete_backup;
 pub mod get_backup;
-pub mod get_backup_keys;
 pub mod get_latest_backup;
 pub mod update_backup;
 
+pub mod add_backup_key_session; // PUT /keys/<roomid>/<sessionid>
```

Comment:
Why are these in a separate declaration group and have comments?

Reply:
The split into two groups was on purpose. The first group manipulates the backup metadata and the second group is a collection of endpoints that manipulate backup content. I added comments because there are so many endpoints with only small differences and it helped me keep track of them better.

Reply:
> The split into two groups was on purpose. The first group manipulates the backup metadata and the second group is a collection of endpoints that manipulate backup content.

Okay. A comment above each group to make this distinction explicit would be good.

> I added comments because there are so many endpoints with only small differences and it helped me keep track of them better.

I understand, but unless you plan to do this for all endpoint module declarations, please remove these again. I'd like to keep them consistent.